lib/libc/minimal/source/stdlib/rand.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2021 Space Cubics, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdlib.h>
+#include <sys/libc-hooks.h>
+#include <app_memory/app_memdomain.h>
+
+#define LIBC_DATA K_APP_DMEM(z_libc_partition)
```

Comment:
@stephanosio, is this the right way to specify the global variable in minimal libc?

Reply:
Yes. `z_libc_partition` is valid as long as `CONFIG_NEED_LIBC_MEM_PARTITION=y`.

Reply:
Thanks.